tasks/run.mjs

```diff
@@ -12,10 +12,15 @@ export async function npmScript (name, args = []) {
   const command = process.platform === 'win32' ? 'npm.cmd' : 'npm'
 
   return new Promise((resolve, reject) => {
-    const script = spawn(command, ['run', name, ...args])
+    const script = spawn(command, ['run', name, '--silent', ...args])
 
+    // Send output to console
     script.stdout.on('data', (data) => console.log(data.toString()))
-    script.stderr.on('data', (data) => console.error(data.toString()))
+
+    // Emit errors to error listener
+    script.stderr.on('data', (data) => {
+      script.emit('error', new Error(data.toString()))
```

Comment:
By emitting an error here it ensures we hit `reject()` (below) via the error listener

Reply:
**question**: Won't that trigger multiple times if a script outputs to `stderr` multiple times? Furthermore, I'm worried:
- it'll catch false errors for a script that just outputs to stderr as a convenient second buffer. I thought the actual marker of a script erroring was a non-zero exit (covered by the `close`)
- it'll double up with the `close` handling later

**question**: Why are we emitting an error rather than directly rejecting? That feels an unnecessary extra step there.

Reply:
>Won't that trigger multiple times if a script outputs to stderr multiple times?

The first error written will hit `script.on('error', reject)` so you'll only see the first one